### PR TITLE
Address minor issues from #5605

### PR DESCRIPTION
**Pull request by nuttycom**

### CODE REVIEW

File: src/wallet/rpcdump.cpp [hunk 1]
```diff
@@ -821,7 +822,7 @@ UniValue z_importviewingkey(const UniValue& params, bool fHelp)
             "1. \"vkey\"             (string, required) The viewing key (see z_exportviewingkey)\n"
             "2. rescan             (string, optional, default=\"whenkeyisnew\") Rescan the wallet for transactions - can be \"yes\", \"no\" or \"whenkeyisnew\"\n"
             "3. startHeight        (numeric, optional, default=0) Block height to start rescan from\n"
-            "\nNote: This call can take minutes to complete if rescan is true.\n"
+            "\nNote: This call can take a long time to complete if rescan is true. Import of Unified viewing keys is not yet supported.\n"
```
Comment (steven-ecc) [inline src/wallet/rpcdump.cpp:825]: Do we plan to support UVK import in the future?

**Review by str4d — APPROVED** [review]
utACK

File: src/wallet/rpcdump.cpp [hunk 2]
```diff
@@ -989,8 +990,10 @@ UniValue z_exportviewingkey(const UniValue& params, bool fHelp)
     if (fHelp || params.size() != 1)
         throw runtime_error(
             "z_exportviewingkey \"zaddr\"\n"
-            "\nReveals the viewing key corresponding to 'zaddr'.\n"
-            "Then the z_importviewingkey can be used with this output\n"
+            "\nReturns the full viewing key corresponding to 'zaddr' in the format specified by\n"
+            "https://zips.z.cash/protocol/protocol.pdf#addressandkeyencoding.\n"
+            "z_importviewingkey can be used to import Sprout and Sapling exported in this format;\n"
+            "import of unified viewing keys is not yet supported.\n"
```
Comment (steven-ecc) [inline src/wallet/rpcdump.cpp:996]: For consistency, should this be "Unified viewing..." to be consistent with line 825 for example?

**Review by steven-ecc — APPROVED** [review]
utACK

File: src/wallet/rpcdump.cpp (hunk 1 above)
Comment (daira) [reply src/wallet/rpcdump.cpp:825]: I think so. This is #5687.

**Review by daira — APPROVED** [review]
utACK (@nuttycom and I paired on this PR, but I've just re-reviewed it).

**Pull request merged**